developers/guides/getstarted/introduction-to-durable-nonces.md

```diff
@@ -0,0 +1,485 @@
+```
+
+Output
+```console
+Error: Hash has expired F13BkBgNTyyuruUQFSgUkXPMJCfPvKhhrr217eiqGfVE
+```
+
+The transfer is not successful because the hash has expired. How do we overcome this issue of expired blockhashes? Using Durable Nonces!
+
+### Using Durable Nonces
+We will use the `nonce-account.json` and `nonce-authority.json` keypairs that we created earlier. We already have a nonce initialized in the `nonce-account`. Let's advance it to get a new one first, just to be sure that the `nonce` isn't already used.
+
+
+```console
+solana new-nonce nonce-account.json
+solana nonce-account nonce-account.json
+```
+
+Output
+```console
+Signature: 3z1sSU7fmdRoBZynVLiJEqa97Ja481nb3r1mLu8buAgwMnaKdF4ZaiBkzrLjPRzn1HV2rh4AHQTJHAQ3DsDiYVpF
+
+Balance: 0.0014999 SOL
+Minimum Balance Required: 0.00144768 SOL
+Nonce blockhash: HNUi6La2QpGJdfcAR6yFFmdgYoCvFZREkve2haMBxXVz
+Fee: 5000 lamports per signature
+Authority: 5CZKcm6PakaRWGK8NogzXvj8CjA71uSofKLohoNi4Wom
+```
+
+Perfect, now let's start with offline co-signing the transaction with `co-signer`'s wallet, but this time, we'll use the `Nonce blockhash` printed above, which is basically the `nonce` stored in the `nonce-account` as the blockhash for the transfer transaction.
+
+```console
+solana transfer receiver.json 0.1 \
+  --sign-only \
+  --nonce nonce-account.json \
+  --blockhash HNUi6La2QpGJdfcAR6yFFmdgYoCvFZREkve2haMBxXVz \
+  --fee-payer co-sender.json \
+  --from H8BHbivzT4DtJxL4J4X53CgnqzTUAEJfptSaEHsCvg51 \
+  --keypair co-sender.json
+```
+
+Output
+```console
+Blockhash: HNUi6La2QpGJdfcAR6yFFmdgYoCvFZREkve2haMBxXVz
+Signers (Pubkey=Signature):
+ HDx43xY4piU3xMxNyRQkj89cqiF15hz5FVW9ergTtZ7S=5tfuPxsXchbVFU745658nsQr5Gqhb5nRnZKLnnovJ2PZBHbqUbe7oB5kDbnq7tjeJ2V8Mywa4gujUjT4BWKRcAdi
+Absent Signers (Pubkey):
+ H8BHbivzT4DtJxL4J4X53CgnqzTUAEJfptSaEHsCvg51
+```
+
+This is very similar to the one we signed using the recent blockhash. Now we'll sign and send the transaction with the `sender`'s wallet.
+
+```console
+solana transfer receiver.json 0.1 \
+  --nonce nonce-account.json \
+  --nonce-authority nonce-authority.json \
+  --blockhash HNUi6La2QpGJdfcAR6yFFmdgYoCvFZREkve2haMBxXVz \
+  --from sender.json \
+  --keypair sender.json \
+  --signer HDx43xY4piU3xMxNyRQkj89cqiF15hz5FVW9ergTtZ7S=5tfuPxsXchbVFU745658nsQr5Gqhb5nRnZKLnnovJ2PZBHbqUbe7oB5kDbnq7tjeJ2V8Mywa4gujUjT4BWKRcAdi
+```
+
+Output
+```console
+Signature: anQ8VtQgeSMoKTnQCubTenq1J7WKxAa1dbFMDLsbDWgV6GGL135G1Ydv4QTNd6GptP3TxDQ2ZWi3Y5qnEtjM7yg
+```
+
+The transaction is successfully submitted!
+
+If we check it on the [explorer](https://solscan.io/tx/anQ8VtQgeSMoKTnQCubTenq1J7WKxAa1dbFMDLsbDWgV6GGL135G1Ydv4QTNd6GptP3TxDQ2ZWi3Y5qnEtjM7yg?cluster=devnet), we can see that an instruction, `AdvanceNonce` was prepended to the transaction, as we discussed before. This is done to avoid using the same nonce again.
```

Comment:
```suggestion
If we check it on the [explorer](https://solscan.io/tx/anQ8VtQgeSMoKTnQCubTenq1J7WKxAa1dbFMDLsbDWgV6GGL135G1Ydv4QTNd6GptP3TxDQ2ZWi3Y5qnEtjM7yg?cluster=devnet), we can see that an `AdvanceNonce` instruction was prepended to the transaction, as we discussed before. This is done to avoid using the same nonce again.
```